src/stateToHTML.js

```diff
@@ -103,6 +142,73 @@ function getWrapperTag(blockType: string): ?string {
   }
 }
 
+function getStyleMap(customMap: ?StyleMap): [StyleMap, Array<string>] {
+  if (customMap == null) {
+    return [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER];
+  }
+  let styleMap = {};
+  let styleOrder = [...DEFAULT_STYLE_ORDER];
+  for (let styleName of Object.keys(customMap)) {
+    if (DEFAULT_STYLE_MAP.hasOwnProperty(styleName)) {
+      let defaultStyles = DEFAULT_STYLE_MAP[styleName];
+      styleMap[styleName] = {...defaultStyles, ...customMap[styleName]};
+    } else {
+      styleMap[styleName] = customMap[styleName];
+      styleOrder.push(styleName);
+    }
+  }
+  return [styleMap, styleOrder];
+}
+
+function styleToCSS(styleDescr: StyleDescr): string {
+  let parts = Object.keys(styleDescr).map((name) => {
```

Comment:
It's getting busy in here :)
These transform functions (`styleToCSS`, `processStyleName|Value`, `normalizeAttributes`) could be extracted to their own modules for ease of reading and testing.